credentials/src/main/kotlin/web5/sdk/credentials/VerifiableCredential.kt

```diff
@@ -209,58 +220,9 @@ public class VerifiableCredential internal constructor(public val vcDataModel: V
      * ```
      */
     public fun verify(vcJwt: String) {
-      val decodedJwt = Jwt.decode(vcJwt)
```

Comment:
this function is a big win (obv the actual verification logic needs to be audited such that we aren't losing functionality, but this is a spike so doesn't matter right now)